application/src/main/java/org/togetherjava/tjbot/commands/moderation/AuditCommand.java

```diff
@@ -32,12 +39,15 @@ public final class AuditCommand extends SlashCommandAdapter {
     private static final String TARGET_OPTION = "user";
     private static final String COMMAND_NAME = "audit";
     private static final String ACTION_VERB = "audit";
+    private static final int MAX_PAGE_LENGTH = 25;
+    private static final String PREVIOUS_BUTTON = "⬅";
+    private static final String NEXT_BUTTON = "➡";
```

Comment:
The names can be confusing. Those are not the buttons but their labels. Add a `_LABEL` suffix to the names please.

Its especially confusing in ur lines `Button nextButton = createButton(NEXT_BUTTON, ...)`, as in having two variables with essentially the same name.